models/fixtures/access.yml

```diff
@@ -1,173 +1,185 @@
 -
   id: 1
+  user_id: 1
+  repo_id: 63
+  mode: 4
+
+-
+  id: 2
   user_id: 2
   repo_id: 3
   mode: 4
 
 -
-  id: 2
+  id: 3
   user_id: 2
   repo_id: 5
   mode: 4
 
 -
-  id: 3
+  id: 4
```

Comment:
Still a lot of unnecessary changes.

Reply:
WIP

Reply:
https://github.com/go-gitea/gitea/pull/25812/commits/9fb523c6892bfb0f5a67a1470070452b4d8e76e1